site/book/01-getting-started/02-quickstart.md

```diff
@@ -8,10 +8,16 @@ configuration using the underlying Git version control system.
 First, let's fetch the _kpt package_ from Git to your local filesystem:
 
 ```shell
-$ kpt pkg get https://github.com/GoogleContainerTools/kpt/package-examples/nginx@next
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt/package-examples/nginx@v0.1
 $ cd nginx
 ```
 
+Commit the local package to `git`:
```

Comment:
nit: Currently, we use Git (not quotated) everywhere. That's how the Git site refers to it.

Reply:
Updated.